subsys/bluetooth/mesh/sensor.c

```diff
@@ -554,6 +554,14 @@ void sensor_cadence_update(struct bt_mesh_sensor *sensor,
 
 	new = sensor_cadence(&sensor->state.threshold, value);
 
+	/** Use Fast Cadence Period Divisor when publishing when the change exceeds the delta,
+	 * section 4.3.1.2.4.3, E15551 and E15886.
+	 */
+	if (new == BT_MESH_SENSOR_CADENCE_NORMAL) {
+		new = bt_mesh_sensor_delta_threshold(sensor, value) ?
+			BT_MESH_SENSOR_CADENCE_FAST : BT_MESH_SENSOR_CADENCE_NORMAL;
+	}
```

Comment:
Could this be moved inside `sensor_cadence` instead?

Reply:
I would keep it here because `sensor_cadence` function is about checking if a value is within a fast cadence interval.